example/flaskr/templates/guest/portal.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'base.html' %}
+
+{% block header %}
+  <h1>{% block title %}BetterBytes{% endblock %}</h1>
+{% endblock %}
+
+{% block content %}
+<h3>Find out how to make more ethical recipes!</h3>
+
+<h3>How does it work?</h3>
+<h4>
+  Choosing common recipes or building a new one, we'll help you learn more about
+  the ethical impacts of the foods we eat!  Common ingredients are listed to include
+  our ethical rating system to provide you the information and alternatives to 
+  design ethical recipes.
+</h4>
+<h4>
+  Explore the application without login requires, and see what ethical impact you can make:
```

Comment:
I think a simpler explanation is fine here since the buttons explain the different routes.
```suggestion
  Explore the application and see what ethical impact you can make:
```